src/domains/community/comment/components/CommentForm.jsx

```
@@ -2,12 +2,13 @@ import { useState, useEffect } from 'react'
 import useAuthStore from '../../../../store/login/useAuthStore'
 import { postComment } from '../../service/commentService'
 import ToastService from '../../../../utils/toastService'
+import useBestStore from '../../../../store/main/useBestStore'
 
 const CommentForm = ({ boardId, onCommentAdded }) => {
```

ESLint

[error] 'boardId' is missing in props validation

(react/prop-types)